src/components/Status/Status.js

```diff
@@ -19,14 +20,16 @@ Status.propTypes = {
   children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
 };
 
-export const OverlayStatus = ({ icon, text, overlay }) => (
-  <Status icon={icon}>
-    <OverlayTrigger overlay={overlay} placement="right" trigger={['click']} rootClose>
-      <Button className="kubevirt-status__button" bsStyle="link">
-        {text}
-      </Button>
-    </OverlayTrigger>
-  </Status>
+export const OverlayStatus = ({ icon, header, children }) => (
```

Comment:
we could rename this to Popover status now

Reply:
Make sense :-)